HW15-refactofing/src/main/java/org/example/BasicMathLibrary.java

```diff
@@ -0,0 +1,27 @@
+package org.example;
+
+public class BasicMathLibrary {
```

Comment:
The logic (or, let's say, implementation) looks good, but the structure is not good :) 
`public` modificator allow access to this class for you lib users - this is something that should be avioded